src/main/java/org/mqttbee/api/mqtt/mqtt5/message/publish/Mqtt5PublishBuilder.java

```diff
@@ -58,7 +57,7 @@
     MqttUTF8StringImpl contentType;
     MqttTopicImpl responseTopic;
     ByteBuffer correlationData;
-    private TopicAliasUsage topicAliasUsage = DEFAULT_TOPIC_ALIAS_USAGE;
+    private TopicAliasUsage useTopicAlias = DEFAULT_TOPIC_ALIAS_USAGE;
```

Comment:
Minor: the attribute and parameter should stay topicAliasUsage

Reply:
DONE